### PR TITLE
[70] [Chore] Add UI test for Complete Survey Flow

### LINKED ISSUE #70: [Chore] Add UI test for Complete Survey Flow
## Why

- Complete Survey Flow needs UI tests
 
## Acceptance Criteria

- Add test functions for Complete Survey Flow
- Tests passed

### PR DESCRIPTION
**Pull request by chiiida**
https://github.com/llleyelll/ic-surveys-ios/issues/70

## What happened 👀

- Add UI test for Complete Survey Flow
 
## Insight 📝

- Update `SeededAPI` to handle `EmptyResponse`
- Update `fetchSurveyDetailSuccess` json data.
- Update some unit tests that using `fetchSurveyDetailSuccess`.
 
## Proof Of Work 📹

Screen Shot 2021-07-14 at 4 41 44 PM

### CODE REVIEW

**Review by markgravity — APPROVED** [review]
LGTM 🤖

File: SurveyApp/Views/AnswerView/TextFieldAnswerView.swift [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //  Created by Chananchida F. on 7/6/21.
 //
 
+import Foundation
```
Comment (blyscuit) [inline SurveyApp/Views/AnswerView/TextFieldAnswerView.swift]: UIKit contains `Foundation`

File: SurveyApp/Views/AnswerView/ChoiceAnswerView.swift [hunk 2]
```diff
@@ -5,6 +5,7 @@
 //  Created by Chananchida F. on 7/6/21.
 //
 
+import Foundation
```
Comment (blyscuit) [inline SurveyApp/Views/AnswerView/ChoiceAnswerView.swift]:
```suggestion

```

File: SurveyApp/Views/Lottie/CompleteSurveyView.swift [hunk 3]
```diff
@@ -5,6 +5,7 @@
 //  Created by Chananchida F. on 7/11/21.
 //
 
+import Foundation
```
Comment (blyscuit) [inline SurveyApp/Views/Lottie/CompleteSurveyView.swift]:
```suggestion

```

**Review by blyscuit — APPROVED** [review]

**Pull request merged**